scripts/Packages/Package_DFN_QFN/size_definitions/qfn_texas.yaml

```diff
@@ -310,3 +310,55 @@ Texas_S-PVQFN-N64_EP:
   #pad_length_addition: 0.5
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
+
+Texas_VQFN_RGE0024H:
+  device_type: 'QFN'
+  manufacturer: 'Texas'
+  #part_number: 'mpn'
+  size_source: 'http://www.ti.com/lit/ds/symlink/tlc5971.pdf#page=42&zoom=200,-13,779'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  custom_name_format: 'Texas_RGE0024H{vias:s}'
```

Comment:
I would suggest to change the name format to:
custom_name_format: 'Texas_RGE00{pincount}H_EP{ep_size_x:g}x{ep_size_y:g}mm{vias:s}'

Reply:
thanks for your feedback.
iam happy with integrating the EP size..
currently there are other TI parts that also lag it.. 
and i don't know if it is something that can change regarding the `RGE`prefix

for the same i have leaved out the pincount  - i found only results with 24 pin for RGE... so to me it seems this is a fixed term...

but as told iam happy changing it..

Reply:
I would really suggest including the EP size for new footprints. (I recently changed the wording of the KLC rules to make this more clear.)
All script generated TI packages include it. I have not yet come around to rework all qfn footprints. Some of the still missing footprints are included in two larger pull requests of mine. (Adding the EP size to the existing footprint names might also require changes on the symbol side. This is why i did not yet finish all of them as it is quite easy to generate merge conflicts if the changes are too big.)